instant-distance/src/lib.rs

```diff
@@ -128,17 +136,20 @@ impl Default for Heuristic {
 }
 
 #[cfg_attr(feature = "serde", derive(Deserialize, Serialize))]
-pub struct HnswMap<P, V> {
-    hnsw: Hnsw<P>,
+pub struct HnswMap<P, V>
+where
```

Comment:
I don't think the bound here is necessary/helpful?

Reply:
Leftovers from more exploratory changes, missed it.